src/values/AbstractValue.js

```diff
@@ -146,7 +146,9 @@ export default class AbstractValue extends Value {
     function add_intrinsic(name: string) {
       if (name.startsWith("_$")) {
         if (gen === undefined) return;
-        add_args(gen.derivedIds.get(name));
+        let temporalBuildNodeEntryArgs = gen.derivedIds.get(name);
+        if (temporalBuildNodeEntryArgs === undefined) return;
```

Comment:
What is this about? Can this happen? What did happen previously in this case?

Reply:
I believe it's to help Flow, as per the usual code. I don't understand how this could ever happen, it doesn't happen when running on our internal bundle and tests at least.

Reply:
Let's change it to an invariant then?

I thought you introduced this code (I didn't see it before), do you mean it's from somewhere else?